src/containers/Footer.js

```diff
@@ -2,23 +2,32 @@ import React from 'react';
 import { connect } from 'react-redux';
 import { AppFooter } from '@coreui/react';
 
-
 export function Footer(props) {
 
 	let FooterService = props.app.locateService("FooterService");
 
 	return (
 		<AppFooter>
-
 			{FooterService.Items.map((item, idx) => (
 				<item.component key={idx} {...item.componentProps} app={props.app}/>
 			))}
 
 			<span className="ml-auto">
+				{ props.footer_image.secondImage &&
+					<>
```

Comment:
<></> is a shorter version of React.Fragment. Since anchor and separator are siblings and inside curly brackets, React need them to be wraped inside some parent. So I've decided that Fragment would be most suitable for that case, because after rendering there won't be any <></> in DOM at all.

https://reactjs.org/docs/react-api.html#reactfragment